src/openrct2/actions/PlaceParkEntranceAction.cpp

```diff
@@ -27,6 +27,12 @@ PlaceParkEntranceAction::PlaceParkEntranceAction(const CoordsXYZD& location, Obj
 {
 }
 
+void PlaceParkEntranceAction::AcceptParameters(GameActionParameterVisitor& visitor)
+{
+    visitor.Visit(_loc);
+    visitor.Visit("pathType", _pathType);
```

Comment:
`pathObject`

Reply:
Done.

Reply:
I had a closer look at this. For FootpathPlace, we use `object` for the surface object, which has legacy reasons as far as I remember, and `railingsObject` for the railings object. All other objects are just `object`.
The entrance tile element's path surface object is accessible as `footpathSurfaceObject`. So, for consistency, I could argue for any of `object`, `surfaceObject`, `footpathObject` or `footpathSurfaceObject`. I decided to go with the latter to match the property name of the entrance tile element.